### ISSUE TITLE
Some problems when running your code

### ISSUE THREAD

**Issue by GResNetReader** [anonymous-sourcecode/GResNet#1]
Hey,
Thanks for your code. 

I believe that Table 2 in your paper was generated with this code, so I run your code with `python script.py` on my local machine to verify those performances reported in your paper. Several problems arose during this process.
![7050b70255825a72935106e68c4529b](https://user-images.githubusercontent.com/58282396/69777415-26854680-11db-11ea-858f-717fd71898fe.png)

1. Do you provide the optimal hyperparameters' setting in this GitHub repository? My experiment results show that 3 layer GAT with graph-naive residual connection could only achieve 82.5% accuracy on Cora test data, which I suppose is far less than 85.5% as you reported. This problem also holds for 5 layer GCN with graph-raw residual connection but I didn't record the result.
![6b981acae09271f7f03665f20395f45](https://user-images.githubusercontent.com/58282396/69777357-e45c0500-11da-11ea-8a72-90f47c824b78.png)
2. I could not run your code on GPU when I set `cuda_tag = True`. Pytorch would throw an error. This makes evaluation of your code for GAT extremely slow.
3. I noticed that the Cora dataset you have used is released by LINQS, which may have a different test-evaluation-test partition from which used in GCN(I am not sure whether you have performed an alignment on the dataset you put in this reprository).

**Comment by anonymous-sourcecode** [comment]
Hi,

Thanks for your interest in our work. 

Regarding your question:

1. Please choose a good epoch to do an early stop, the acc reported at the end of your screenshot is the acc achieve at 500th epoch. We report the best encountered acc in Table 2 in the paper (not necessarily the one obtained at the 500th epoch).

2. For the GAT model, it is very slow. We use the source code from https://github.com/Diego999/pyGAT

3. For the Cora data, we use the one from https://github.com/tkipf/pygcn

Just FYI.

**Comment by liu6zijian** [comment]
Dear researcher,
Thanks for your great code.
I run your code cloned from your github source, but I find there is a gap between result we run and your provide in the paper. We absolutely do not change the code (model architecture, random seed or hyper parameters).
For example, my experiment results show that 5 layers GCN with graph-naive residual connection could only achieve 80.2% accuracy on Cora test data, which I suppose is far less than 84.3% as you provided. Are there some tricks? Thank you.
![image](https://user-images.githubusercontent.com/30034194/93304620-df327a00-f82f-11ea-9664-77a2c15855a6.png)

**Comment by anonymous-sourcecode** [comment]
1. Do an early stop, you don’t need to run 1000 epoch. The score you reported is at the 1000th epoch. Please choose a better epoch number.
2. Change another random seed to get a new initialization of the parameters.
